### ISSUE TITLE
识别单张png/jpg用时1分钟左右，似乎gpu没有被用起来并且报了一大堆警告信息，向大佬请教，感谢！！

### ISSUE THREAD

**Issue by B3DDD** [deepseek-ai/DeepSeek-OCR#286]
1.在本地笔记本conda（py3.12)部署了deepseek-ocr模型，
gpu是rtx4060 laptop 8GB，
cpu是intel core ULTRA9
环境是：Name: transformers Version: 4.46.3
Name: torch Version: 2.7.1+cu128
Name: tokenizers Version: 0.20.3
Name: flash_attn Version: 2.8.2
Cuda compilation tools, release 12.8, V12.8.61

2.OCR识别能用，transformer脚本推理的，提示词模板功能均能正常使用，就是比较慢gandam模式下每张图都在1分钟左右，并且伴有以下大段警告信息。猜测是gpu没有被用起来，比如为了规避oom或者flash-atten没有被正常启用之类的原因。个人持续在探索中，同时在这里挂个问题，向大佬请教,非常感谢！

3.脚本执行的警告信息：
”You are using a model of type deepseek_vl_v2 to instantiate a model of type DeepseekOCR. This is not supported for all configurations of models and can yield errors.
You are attempting to use Flash Attention 2.0 with a model not initialized on GPU. Make sure to move the model to GPU after initializing it on CPU with `model.to('cuda')`.
Some weights of DeepseekOCRForCausalLM were not initialized from the model checkpoint at deepseek-ai/DeepSeek-OCR and are newly initialized: ['model.vision_model.embeddings.position_ids']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
<class 'transformers_modules.deepseek-ai.DeepSeek-OCR.9f30c71f441d010e5429c532364a86705536c53a.modeling_deepseekocr.DeepseekOCRForCausalLM'>
C:\Users\drbou\AppData\Roaming\Python\Python312\site-packages\transformers\generation\configuration_utils.py:590: UserWarning: `do_sample` is set to `False`. However, `temperature` is set to `0.0` -- this flag is only used in sample-based generation modes. You should set `do_sample=True` or unset `temperature`.
  warnings.warn(
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:None for open-end generation.
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
The `seen_tokens` attribute is deprecated and will be removed in v4.41. Use the `cache_position` model input instead.
`get_max_cache()` is deprecated for all Cache classes. Use `get_max_cache_shape()` instead. Calling `get_max_cache()` will raise error from v4.48“
光是出这段警告信息都花了半分多钟，这段警告信息输出完后才开始输出识别结果。识别结果的输出算是比较快的，10token/s左右的速度。
